validation.js

```diff
@@ -0,0 +1,148 @@
+console.log('validation.js loaded check');
+document.addEventListener('DOMContentLoaded', function () {
+    const emailLogin = document.getElementById('email-login');
```

Comment:
❗️ 수정요청
변수명은 가독성에 있어서 중요한 역할을 합니다. 따라서 어떤 값인지 알 수 있는 변수명을 추천드려요.
아마 로그인과 회원가입에서 같은 js 파일을 쓰시느라 이렇게 이름을 지으신 것 같은데 다른 코멘트에서 말씀드린 것처럼 파일을 분리하시고 변수명은 email-login이라는 클래스명을 가진 input 태그이니 emailInput 과 같은 이름으로 변경하시는 것을 추천드립니다.

```suggestion
    const emailInput = document.getElementById('email-login');
```